lib/model.js

```diff
@@ -1404,6 +1410,18 @@ Model.prototype.findAll = function(options) {
       return this.runHooks('afterFind', results, options);
     }
   }).then(function (results) {
+
+    //kenophobic mode
+    if(_.isEmpty(results) && options.kenophobic) {
+      if( (typeof options.kenophobic === 'function')) {
```

Comment:
space after `if` (goes for entire diff), no space after `(`, no need to have double `()` here actually.